frontend/config-assistant/src/components/toolbar/GenerateFileClearance.ts

```diff
@@ -0,0 +1,48 @@
+import {ChangeResponsible, useSessionStore} from '@/store/sessionStore';
+import {toastService} from '@/helpers/toastService';
+import {confirmationService} from '@/helpers/confirmationService';
+import {useDataStore} from '@/store/dataStore';
+import {generateSampleData} from '@/components/toolbar/createSampleData';
+import {errorService} from '@/main';
+
+export function newEmptyFileAfterGeneration(message: string | undefined = undefined): void {
+  if (!message) {
+    clearEditorGeneration();
+    return;
+  }
+  confirmationService.require({
+    message: message,
+    header: 'Delete Confirmation',
+    icon: 'pi pi-info-circle',
+    acceptClass: 'p-button-danger',
+    accept: () => {
+      toastService.add({
+        severity: 'info',
+        summary: 'Confirmed',
+        detail: 'Config data is deleted from File editor',
+        life: 3000,
+      });
+      clearEditorGeneration();
+    },
+    reject: () => {
+      toastService.add({
```

Comment:
I would remove this toast. We should also remove in the other menus, it's very confusing